lib/mutant/cli/command/environment.rb

```diff
@@ -103,10 +103,9 @@ def add_matcher_options(parser)
             add_matcher(:start_expressions, @config.expression_parser.call(pattern).from_right)
           end
           parser.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
-            add_matcher(
-              :subject_filters,
-              Repository::SubjectFilter.new(diff: Repository::Diff.new(to: revision, world: world))
-            )
+            diff = Repository::Diff.new(to: revision, world: world)
+            add_matcher(:subject_filters, Repository::SubjectFilter.new(diff: diff))
+            add_matcher(:diffs, diff)
```

Comment:
Seems slightly odd to pass the same diff to do different places but this was the simplest/smallest change I could think of.

Reply:
Its re-using the same object for different purposes, re-using objects if fine by me.